splinterd/tests/admin/circuit_abandon.rs

```diff
@@ -483,3 +483,336 @@ pub fn test_3_party_circuit_abandon() {
+///    filtered on the circuit's status (`status=abandoned`)
+/// 13. Stop the first node in the network
+/// 14. Create and submit a `CircuitAbandon` payload to completely abandon the circuit for all
+///    members, from the third node in the network
+/// 15. Restart the stopped node
+/// 16. Verify the circuit is returned as `Abandoned` for each node, using `list_circuits` filtered
+///     on the circuit's status (`status=abandoned`)
+#[test]
+#[ignore]
+pub fn test_3_party_circuit_abandon_stop() {
+    // Start a 2-node network
+    let mut network = Network::new()
+        .with_default_rest_api_variant(RestApiVariant::ActixWeb1)
+        .add_nodes_with_defaults(3)
+        .expect("Unable to start 3-node ActixWeb1 network");
+    // Get the first node in the network
+    let mut node_a = network.node(0).expect("Unable to get first node");
+    // Get the second node in the network
+    let mut node_b = network.node(1).expect("Unable to get second node");
+    // Get the third node from the network
+    let mut node_c = network.node(2).expect("Unable to get third node");
+
+    let circuit_id = "ABCDE-01234";
+    // Commit a circuit to state
+    commit_3_party_circuit(&circuit_id, node_a, node_b, node_c);
+    // Commit a circuit between the nodes that will remain active while the other circuit is
+    // abandoned
+    let active_circuit_id = "FGHIJ-56789";
+    // Commit the circuit to state
+    commit_3_party_circuit(&active_circuit_id, node_a, node_b, node_c);
+
+    // Stop the second node in the network
+    network = network.stop(1).expect("Unable to stop second node");
+    node_a = network.node(0).expect("Unable to get first node");
+
+    // Create the abandon request to be sent from the first node
+    let abandon_payload = make_circuit_abandon_payload(
+        &circuit_id,
+        node_a.node_id(),
+        &*node_a.admin_signer().clone_box(),
+    );
+    assert!(node_a
+        .admin_service_client()
+        .submit_admin_payload(abandon_payload)
+        .is_ok());
+
+    // Restart the second node in the network
+    network = network.start(1).expect("Unable to start second node");
+    node_a = network.node(0).expect("Unable to get first node");
+    node_b = network.node(1).expect("Unable to get second node");
+    node_c = network.node(2).expect("Unable to get third node");
+
+    assert_eq!(
+        node_a
+            .admin_service_client()
+            .list_circuits(Some("status=abandoned"))
+            .expect("Unable to list circuits")
+            .data
+            .len(),
+        1
+    );
+    assert_eq!(
+        node_b
+            .admin_service_client()
+            .list_circuits(None)
+            .expect("Unable to list circuits")
+            .data
+            .len(),
+        2
+    );
+    assert_eq!(
+        node_c
+            .admin_service_client()
+            .list_circuits(None)
+            .expect("Unable to list_circuits")
```

Comment:
"list_circuits" should probably be "list circuits" to match the other panic messages, there are a few other panic messages below this line in this test that use "list_circuits" as well

Reply:
Fixed the few (3) instances of this typo